clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -90,14 +90,15 @@ object UncommittedGarbageCollector {
     val cutoffTime = DateUtils.addSeconds(new Date(), -minAgeSeconds)
     val startTime = java.time.Clock.systemUTC.instant()
 
-    val shouldMark = hc.getBoolean(LAKEFS_CONF_GC_DO_MARK, true)
-    val shouldSweep = hc.getBoolean(LAKEFS_CONF_GC_DO_SWEEP, true)
+    val shouldMark = hc.getBoolean(LAKEFS_CONF_GC_DO_MARK, false)
```

Comment:
@itaiad200  initially I thought so too - but then I figured that if someone wants to perform only sweep and forgets to set mark to false it will result in an unwanted behavior

Reply:
done